src/telliot_core/contract/contract.py

```diff
@@ -143,10 +143,16 @@ async def write(
             tx_dict = {
                 "from": acc.address,
                 "nonce": acc_nonce,
-                "gas": gas_limit,
-                "chainId": self.node.chain_id,
```

Comment:
Dont' have to pass in chainId anymore to tx dictionary?

Reply:
no you don't. it gets it from the endpoint/web3